### PR TITLE
fix websocket issue

### PR DESCRIPTION
**Pull request by mamantoha**
There is another issue with Kemal in Crystal 0.35.0. Websockets are broken.
I create a repository where you can reproduce an issue https://github.com/mamantoha/kemal_websocket_issue

Run

```
crystal server.cr
```

Open http://localhost:3000

websocket_issue01

### CODE REVIEW

**Comment by xaviablaza** [comment]
Would love to see this merged into the upstream, thank you! @mamantoha

File: src/kemal/websocket_handler.cr [hunk 1]
```diff
@@ -12,8 +12,6 @@ module Kemal
     def call(context : HTTP::Server::Context)
       return call_next(context) unless context.ws_route_found? && websocket_upgrade_request?(context)
       content = context.websocket.call(context)
```
Comment (Sija) [inline src/kemal/websocket_handler.cr:14]:
```suggestion
      context.websocket.call(context)
```

**Comment by msa7** [comment]
What about merge it?

**Comment by drum445** [comment]
Any chance of this being merged/fixed?

**Pull request merged**